tests/5.1/masked/test_masked.c

```diff
@@ -0,0 +1,47 @@
+//===--- test_masked.c ----------------------------------------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+// 
+// This test utilizes the masked construct to ensure that a section of code
+// only runs on a certain thread. When the filter clause is not present, the
+// primary thread runs the structured block.
+//
+//===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int test_masked() {
+  int errors = 0;
+  int i;
+  int x[N];
+  int ct = 0;
+  int total = 10;
+  for(i=0; i<N; i++){
+    x[i] = 0;
```

Comment:
Additionally, x assigned to outside any OpenMP construct but otherwise not used. (I think @spophale did remark this already.)